Schema/V1/no.ks.fiks.arkiv.v1.innsyn.sok.xsd

```diff
@@ -383,7 +383,7 @@
 
     <xs:complexType name="saksmappeSortering">
         <xs:sequence>
-            <xs:element name="felt" type="saksmappeSorteringsfelt"/>
+            <xs:element name="felt" type="mappeOgSaksmappeSorteringsfelt"/>
```

Comment:
Jeg ser ingen grunn til å døpe om disse siden saksmappe uansett er en utvidelse til mappe.

Reply:
Jeg kan endre dette ja. Det ble sånn da jeg brukte `union` for å kunne skille hva som er mappe og saksmappe felt og dermed kunne gjenbruke i stedet for å kopiere inn det som er i mappe til saksmappe simpleType. Men jeg kan gjøre navnet enklere.

Reply:
Hvis det som er saksmappefelt aldri skal brukes direkte er det bedre å la det ha et "odde" navn og så la unionen hete saksmappefelt.

Reply:
Endte opp med å ta bort union. Siden man aldri skal bruke enum som inneholder felter for kun saksmappe så blir det bedre å heller å kopiere verdiene fra mappe til saksmappe.